package.js

```diff
@@ -12,7 +12,7 @@ Package.describe({
 })
 
 Package.onUse(function (api) {
-  api.versionsFrom('1.8.1')
+  api.versionsFrom('1.4')
```

Comment:
I think we should have minimum version set to 1.9 as that is where Node 12 started, so anything before that in my opinion is not worth encouraging.

Reply:
Keep in mind we're going to remove the corresponding meteor release checks, are you ok with that?

Reply:
Yes. Won't that also make the tests faster since they have less versions to test on?

Reply:
Absolutely :+1: